code/game/objects/items/storage/bags.dm

```diff
@@ -209,6 +209,30 @@
 		))
 ////////
 
+// -----------------------------
+//          Mini Plant Bag
+// -----------------------------
+
+/obj/item/storage/bag/smallplants
+	name = "Small plant bag"
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "plantbag"
+	worn_icon_state = "plantbag"
+	resistance_flags = FLAMMABLE
+
+/obj/item/storage/bag/plants/Initialize(mapload)
```

Comment:
this is the plants bag not the small plants bag

Reply:
Also this doesn't have to all be set in the initialize since parent does it all, I think all that needs to be done is editing the slots and stuff.

Reply:
how do i make it not happen in init?